source/git-parser.js

```diff
@@ -336,3 +337,10 @@ exports.parsePatchDiffResult = (patchLineList, text) => {
     return null;
   }
 }
+
+// determine the git directory, always an absolute path
+// TODO: This method does not handle errors, it assumes that repoPath is a valid git repository path
+exports.findGitDir = (repoPath) => {
+  const gitDir = child_process.execSync('git rev-parse --absolute-git-dir', {cwd: repoPath}).toString().trim();
+  return gitDir;
+}
```

Comment:
`git-parser.js` is not the place for executing git commands.  That is within `git-promise.js`.


`git-parser` is exclusively for parsing the git output